samples/matter/template/README.rst

```diff
@@ -58,6 +58,18 @@ As a result, the device can join the Thread network and communicate with other T
 To start the commissioning procedure, the controller must get the commissioning information from the Matter accessory device.
 The data payload, which includes the device discriminator and setup PIN code, is encoded within a QR code, printed to the UART console.
 
+Configuration
+*************
+
+|config|
+
+FEM support
```

Comment:
Are you sure you want to have such a long piece of documentation added to the template sample docs? I think that adding it here also requires changes to the Create a Matter accessory device guide, as we tell the user there to copy the template sample, so it is natural to ask how creating the accessory device impacts the FEM support.

Reply:
After discussing this topic on Teams, I'm fine with adding this FEM support section to the template sample.